test/test_bucket.py

```diff
@@ -512,17 +512,33 @@ def test_download_by_id_progress_range_one_off(self):
                 range_=(0, 11),
             )
 
-    def test_download_by_id_progress_invalid_range(self):
-        with self.assertRaises(
-            InvalidRange,
-            msg='Cloud can only serve a range of 0-10, while a range of 5-16 was requested',
-        ):
+    def test_download_by_id_progress_partial_inplace_overwrite(self):
+        # LOCAL is
+        # 12345678901234567890
+        #
+        # and then:
+        #
+        # hello world
+        #    |||||||
+        #    |||||||
+        #    vvvvvvv
+        #
+        # 123lo worl1234567890
+
+        with TempDir() as d:
+            path = os.path.join(d, 'file2')
+            download_dest = PreSeekedDownloadDest(seek_target=3, local_file_path=path)
```

Comment:
It might be a better test if the local offset an the cloud offset were different.

Reply:
I had two tests there, might have not committed both. I'll check.

Reply:
Ok, I added another test with different offsets on local and cloud side